applications/ConstitutiveLawsApplication/constitutive_laws_application.cpp

```diff
@@ -0,0 +1,37 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:		 BSD License
+//					 Kratos default license: kratos/license.txt
+//
+//  Main authors:    Alejandro Cornejo Velazquez
+//                   Riccardo Rossi
+//
```

Comment:
Header format

Reply:
what do you mean?
I only used the app generator, if it is wrong here then we should also correct it there

Reply:
ah the indentation is what you mean right?

Reply:
Yes. We should probably correct it in there too.